sophiread/.clang-format

```diff
@@ -246,5 +246,3 @@ WhitespaceSensitiveMacros:
   - BOOST_PP_STRINGIZE
   - NS_SWIFT_NAME
   - CF_SWIFT_NAME
-...
```

Comment:
FYI: [mantid's version](https://github.com/mantidproject/mantid/blob/main/.clang-format) is LLVM style with different line length

Reply:
I noticed when I tried to run the pre-commit locally.
Since it will change a lot of files, I will switch in the next PR, along with all the clang-format changes.